net/ocserv/files/ocserv.init

```diff
@@ -33,26 +35,27 @@ setup_config() {
 				uci set dhcp.lan.start=100
 				uci set dhcp.lan.limit=91
 			fi
-			ip=$(uci get network.lan.ipaddr)
+			network_get_ipaddr ip lan
 			ipaddr="$(echo $ip|cut -d . -f1,2,3).192"
 			netmask="255.255.255.192"
-			uci set ocserv.config.ipaddr="$ipaddr"
-			uci set ocserv.config.netmask="$netmask"
-			uci commit
 		fi
 
 		if test -z "$ip6addr";then
-			ip6addr=$(uci get network.lan.ip6addr 2>/dev/null)
-			test -n "$ip6addr" && uci set ocserv.config.ip6addr="$ip6addr"
-			uci commit
+			network_get_ipaddr6 ip6addr lan
+			# Append ipv6 prefix
+			test -n "$ip6addr" && ip6addr="$ip6addr/96"
 		fi
 
 		ping_leases=1
-		test -n "$ipaddr" && sysctl -w "net.ipv4.conf.$(uci get network.lan.ifname).proxy_arp"=1 >/dev/null
-		test -n "$ip6addr" && sysctl -w "net.ipv6.conf.$(uci get network.lan.ifname).proxy_ndp"=1 >/dev/null
+		local ifname
+		network_get_device ifname lan
+		if test -n "ifname";then
```

Comment:
This looks like a typo and should likely be `test -n "$ifname"`.

You can also shorten the code into just

    if network_get_device ifname lan; then
        ...
    fi